index.js

```diff
@@ -65,7 +65,14 @@ BroccoliMergeTrees.prototype.build = function() {
 
   var applyPatchStart = new Date();
 
-  this._applyPatch(patch);
+  try {
+    this._applyPatch(patch);
+  } catch(e) {
+    // Whatever the failure, start again and do a complete build next time
+    this._currentTree = FSTree.fromPaths([]);
```

Comment:
@stefanpenner yeah I don't think we should scan the output to redetermine state unless it's for some known recoverable error.

That said, we should probably output a warning here so users are aware that there's a bug that we should look into.  Do you agree?